gotests_test.go

```diff
@@ -812,6 +813,124 @@ func TestGenerateTests(t *testing.T) {
 			},
 			want: mustReadAndFormatGoFile(t, "testdata/goldens/function_with_return_value_template_data.go"),
 		},
+		{
+			name: "named_named=on",
+			args: args{
+				srcPath: "testdata/test038.go",
+				named:   true,
+			},
+			want: mustReadAndFormatGoFile(t, "testdata/named/named_on.go"),
+		},
+		{
+			name: "named_named=on,template=testify",
+			args: args{
+				srcPath:  "testdata/test038.go",
+				template: "testify",
+				named:    true,
+			},
+			want: mustReadAndFormatGoFile(t, "testdata/named/named_on_template_testify.go"),
+		},
+		{
+			name: "named_named=off",
```

Comment:
Why do we need tests when named=off? Named is off by default, so I think this is already covered by tests above.

Reply:
Did it for consistency's sake (named off/on, subtests=on/off). But you are right, i better to remove this test.